ansible_collections/arista/avd/molecule/digital_twin/inventory/host_vars/digital-twin-adjust-oob-mgmt-4.yml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Please add a short description to each file so we know what we are testing in each. Thanks.